tests/suite_test.go

```diff
@@ -90,19 +73,27 @@ type ClientManager struct {
 	mfopts    mf.Option
 }
 
-// TestAPIs - This is the entry point for Ginkgo -
-// the go test runner will run this function when you run go test or ginkgo.
-// Under the hood, Ginkgo is simply calling go test.
-// You can run go test instead of the ginkgo CLI, But Ginkgo has several capabilities that can only be accessed via ginkgo.
-// It is best practice to embrace the ginkgo CLI and treat it as a first-class member of the testing toolchain.
-func TestAPIs(t *testing.T) {
-	// Single line of glue code connecting Ginkgo to Gomega
-	// Inform our matcher library (Gomega) which function to call (Ginkgo's Fail) in the event a failure is detected.
-	RegisterFailHandler(Fail)
-
-	// Inform Ginkgo to start the test suite, passing it the *testing.T instance and a description of the suite.
-	// Only call RunSpecs once and let Ginkgo worry about calling *testing.T for us.
-	RunSpecs(t, "Controller Suite")
+type IntegrationTestSuite struct {
+	suite.Suite
+	Clientmgr     ClientManager                                 // Note the capitalization to export
+	Ctx           context.Context                               // Exported and added
+	DSPANamespace string                                        // Already exported, make sure it's part of the struct
+	DSPA          *dspav1alpha1.DataSciencePipelinesApplication // Exported and added
```

Comment:
can you explain these comments? exported and added where?

Reply:
Removed these comments, not needed anymore.